streams/src/main/java/org/apache/kafka/streams/state/internals/CachedStateStore.java

```diff
@@ -23,6 +23,6 @@
      * @param listener
      * @param sendOldValues
      */
-    void setFlushListener(final CacheFlushListener<K, V> listener,
-                          final boolean sendOldValues);
+    boolean setFlushListener(final CacheFlushListener<K, V> listener,
```

Comment:
We return `true` if the listener was registered, ie, if the "metered stores" wraps a "caching store".

Reply:
Thanks. _blushing_